### ISSUE TITLE
Idea: web app showing "top 10 deals"

### ISSUE THREAD

**Issue by Mchristos** [JemLukeBingham/used_car_scraper#9]
How about we create a web app that continuously scrapes the latest car data and displays the "top 10 car deals" (according to whatever metric we choose)? Could be a fun learning experience. I want to learn React/Redux for the frontend and then Docker/Kubernetes for deploying stuff.

**Comment by JemLukeBingham** [comment]
Yeah that's an awesome idea. I've also been wanting to learn Docker

**Comment by JemLukeBingham** [comment]
It's going to be a bit slow this week from my side still (just fyi). hope to be free to work on it soon.
